tests/e2e/test.sh

```diff
@@ -89,18 +93,32 @@ if [ "$replicas" -eq 1 ]; then
   exit 1
 fi
 
-echo "Waiting for deployment to scale down back to 0 within 2 minutes"
+# Verify that leader election works by forcing a 120 second apiserver outage
+KIND_NODE=$(kind get nodes --name=substratus-test)
+docker exec ${KIND_NODE} iptables -I INPUT -p tcp --dport 6443 -j DROP
```

Comment:
Is `6443` the port of the API Server running on kind?

Reply:
Yep and credit to chatGPT with coming up with the high-level idea to simply block the traffic. 

```
k describe svc kubernetes 
Name:              kubernetes
Namespace:         default
Labels:            component=apiserver
                   provider=kubernetes
Annotations:       <none>
Selector:          <none>
Type:              ClusterIP
IP Family Policy:  SingleStack
IP Families:       IPv4
IP:                10.96.0.1
IPs:               10.96.0.1
Port:              https  443/TCP
TargetPort:        6443/TCP
Endpoints:         172.18.0.2:6443
Session Affinity:  None
Events:            <none>
```